.travis.yml

```diff
@@ -13,7 +23,9 @@ install:
     - curl -si https://api.github.com/repos/hpcugent/easybuild-framework/git/refs/heads/${BRANCH} | grep sha
```

Comment:
so, you want the log to include something like:

```
https://github.com/hpcugent/easybuild-framework/tree/d9016c2e22a6b0c6eb3763251e8790fb4bdea85e
```

which is basically

```
SHA=`curl -si https://api.github.com/repos/hpcugent/easybuild-framework/git/refs/heads/${BRANCH} | grep sha`
echo "Using easybuild-framework @ https://github.com/hpcugent/easybuild-framework/tree/$SHA"
```

@wpoely86 please open a separate PR for that (also to easyconfigs repo)